javascript/CommentsInterface.js

```diff
@@ -132,16 +132,23 @@
 		$(':input', form).on('change keydown', function() {
 		   previewEl.removeClass('loading').hide();
 		});*/
-		
+
 		/**
 		 * Clicking one of the metalinks performs the operation via ajax
 		 * this inclues the spam and approve links
 		 */
-		/* @todo Migrate to work with nested comments
-		commentsList.on('click', '.action-links a', function(e) {
+
+		$('..comments-holder .comments-list').on('click', 'div.comment-moderation-options a', function(e) {
 			var link = $(this);
+            if (link.hasClass('delete')) {
+                var confirmation = window.confirm("Are you sure?");
```

Comment:
I think we need to localise this; maybe pop the i18n. js api call in for now, and I can add in the actual transifex string integration later.